linker/descriptors.go

```diff
@@ -1759,11 +1830,17 @@ func (m *mtdDescriptor) Options() protoreflect.ProtoMessage {
 }
 
 func (m *mtdDescriptor) Input() protoreflect.MessageDescriptor {
-	return m.file.ResolveMessageType(protoreflect.FullName(m.proto.GetInputType()))
+	if m.inputType == nil {
```

Comment:
Same question here - can we initialize these up front in createMethodDescriptor?

Reply:
Yep, my bad.

This change was from the first commit, before I did a big refactor. And I forgot to come back and update some of these spots. Got 'em now.